### DISCUSSION TITLE
Description field in waypoints

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#401] [Q&A] [answered]
The via trkpt desc is a gpx element that is still free to use in Cruiser.
It can be used to add extra text information in via points.
A short example demo navigation track is attached.
[Desc via example Eifel.gpx.txt](https://github.com/devemux86/cruiser/files/13697827/Desc.via.example.Eifel.gpx.txt)
In Locus map the via point **Biker Ranch** desc info is displayed like next.
![image](https://github.com/devemux86/cruiser/assets/54167201/bad67521-9989-40dc-a4fb-e027f4be4f8b)

**Comment by devemux86** [comment]
@0709wiwiwi thanks for the suggestion.

Additions to the route model will change the route formats, so it is more complicated.

**Comment by devemux86** [comment, marked as answer]
- Implemented in https://github.com/devemux86/cruiser/discussions/891
